### ISSUE TITLE
Multiple Streams from Single User

### ISSUE THREAD

**Issue by warsix4** [Ombrelin/plex-rich-presence#6]
Currently if there are two streams going to multiple devices only one seems to be detected. 

In my situation a Chromecast is playing media and the local computer that the .jar runs on also has the Plex for Windows app playing media. The Chromecast stream is what shows up in the Discord presence for some reason.

**Comment by Ombrelin** [comment]
Yeah for now the first stream in the list is picked up, it would be nice to be able to choose indeed
